src/info/info.js

```diff
@@ -97,32 +97,51 @@ export class Info extends EventSubscriber {
             acquisition_date = new Date(this.image_info.image_timestamp * 1000).toISOString().slice(-25, -14);
         }
         let pixels_size = "";
+        let pixels_size_label = "Pixels Size (XYZ)";
         if (typeof this.image_info.image_pixels_size === 'object') {
+            let symbol = '\xB5m'; // microns by default
+            let unit = 'MICROMETER';
             let count = 0;
+            let set = false;
             if (typeof this.image_info.image_pixels_size.x == 'number') {
```

Comment:
I am not actually checking on the same value (I need to fix that!) since that part has changed